formstack-consents/src/main/resources/logback.xml

```diff
@@ -0,0 +1,13 @@
+<configuration>
+    <contextName>formstack-consents</contextName>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
```

Comment:
With this appender, are multiline log statements considered as individual logs in cloudwatch? If so you can use the [`AwsLambdaAppender`](https://github.com/guardian/identity-processes/blob/master/payment-failure/src/main/resources/logback.xml#L3).

Reply:
Thank you